lib/matplotlib/textpath.py

```diff
@@ -382,7 +383,10 @@ def get_glyphs_tex(self, prop, s, glyph_map=None,
                     charcode = glyph
 
                 if charcode is not None:
-                    glyph0 = font.load_char(charcode, flags=ft2font_flag)
+                    if use_glyph:
```

Comment:
Actually, this is very problematic, the code path above where you set `use_glyph` is a caching mechanism so the second time around this may have the wrong value of `use_glyph`?